### ISSUE TITLE
truth-tables.html: implement the truth tables

### ISSUE THREAD

**Issue by lcabrini** [technojo/technojo-it-learning#5]
depends on #4 

Under each heading, create the truth table for that logic operator.

**Comment by technojo** [comment]
Hello sir i am having problems creating tables with html

**Comment by lcabrini** [comment]
You need three levels of tags: table for the whole table, tr for each row and td for each cell. You can also use th for the header cells.

Here is an example:

```
<table>
  <tr>
    <th>Name</th>
    <th>Phone</th>
  </tr>

  <tr>
    <td>James</td>
    <td>020 - 00 00 00</td>
  </tr>
</table>
```

**Comment by technojo** [comment]
hello Yamo,
you will realize from my code that i have tried creating a sub heading " NOT" but it continues to pop up just below the first header. any solutions?

**Comment by lcabrini** [comment]
You have not closes the table tag, which i noticed is because I made a mistake in the template above. I have fixed it. After the last end tr in the table, you need to also close the table.

**Comment by lcabrini** [comment]
OK, this is done. But the commit message was a bit wrong, so you will have to close this issue manually.

The message should be.

```
created truth tables. closes #5 
```

It must be the word closes followed by the issue number.

Close this issue manually.
